locksmith/package.json

```diff
@@ -7,8 +7,9 @@
     "prod": "cd build && node -r ./src/utils/envLoader.js ./server.js",
     "start": "yarn build && node -r ./src/utils/envLoader.js ./build/server.js",
     "dev": "NODE_ENV=development yarn db:migrate && tsc-watch --onSuccess 'yarn start' --onFailure 'echo Beep! Compilation Failed'",
-    "test": "NODE_ENV=test yarn run db:migrate && NODE_ENV=test jest --forceExit --runInBand",
-    "pretest": "NODE_ENV=test sequelize --config ./config/sequelize.config.js db:drop",
```

Comment:
Ha! I added that yesterday and thought it had fixed things but I now realize it is because the tests were not running :(